x/payment/keeper/stream_record.go

```diff
@@ -107,9 +107,6 @@ func (k Keeper) GetAllStreamRecord(ctx sdk.Context) (list []types.StreamRecord)
 // it only handles the lock balance change and ignore the other changes(since the streams are already changed and the
 // accumulated OutFlows are changed outside this function)
 func (k Keeper) UpdateFrozenStreamRecord(ctx sdk.Context, streamRecord *types.StreamRecord, change *types.StreamRecordChange) error {
-	if streamRecord.Status != types.STREAM_ACCOUNT_STATUS_FROZEN {
```

Comment:
why delete this?

Reply:
when calling this function, the status has been ensured `STREAM_ACCOUNT_STATUS_FROZEN`